src/handler/promote.ts

```diff
@@ -51,43 +53,28 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
 
     logger.debug(`Current result ${JSON.stringify(currentResult)}`);
 
-    const recordsToArchive: TechrecordGet[] = [];
+    const recordsToArchive: TechRecordGet[] = [];
 
     if (currentResult.length > 0) {
-      const currentRecord = await getBySystemNumberAndCreatedTimestamp(currentResult[0].systemNumber, currentResult[0].createdTimestamp);
-      currentRecord.techRecord_statusCode = StatusCode.ARCHIVED;
-      currentRecord.techRecord_lastUpdatedAt = new Date().toISOString();
-      currentRecord.techRecord_lastUpdatedByName = userDetails.username;
-      currentRecord.techRecord_lastUpdatedById = userDetails.msOid;
-      recordsToArchive.push(currentRecord);
-
-      logger.debug(`Old current record after update ${JSON.stringify(currentRecord)}`);
+      const currentRecord = await getBySystemNumberAndCreatedTimestamp(currentResult[0].systemNumber, currentResult[0].createdTimestamp) as TechRecordGet;
```

Comment:
I know you didn't (right now) write this, but can we use `.find()` to find the current technical record (on line 52) instead of `.filter()` ? We're only using `[0]` anyways